xbundle/__init__.py

```diff
@@ -21,10 +21,10 @@
 import string
 import glob
 import subprocess
+from os.path import join, exists
 
 from lxml import etree
 from lxml.html.soupparser import fromstring as fsbs
-from path import path	# needs path.py
```

Comment:
It's an extra, unnecessary dependency. It's trivial to use the standard library to do the same thing.